### PR TITLE
Add gettext string extraction

### PR DESCRIPTION
**Pull request by mzazrivec**
* add react-gettext-parser to extract i18n strings (plus config and friends)
* add missing gettext annotations
* add catalog `locale/react-ui-components.pot` (with extracted strings)

### CODE REVIEW

**Comment by mzazrivec** [comment]
@himdel review please?

File: package.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
     "build-storybook": "build-storybook -c scripts/storybook -o .out",
     "coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
     "prepublishOnly": "npm run build",
-    "release": "semantic-release"
+    "release": "semantic-release",
+    "gettext-extract": "yarn run react-gettext-parser --config ./scripts/react-gettext-parser-config.js 'src/**/{*.js,*.jsx}'"
```
Comment (himdel) [inline package.json]: in ui-components, the task is called `gettext:extract` (colon vs dash), is that an intentional difference?

File: scripts/react-gettext-parser-config.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+exports.output = 'locale/react-ui-components.pot';
+exports.funcArgumentsMap = { __: ['msgid'] };
```
Comment (himdel) [inline scripts/react-gettext-parser-config.js]: ui-classic is also exposing `n__` , should we do the same here?

**Comment by himdel** [comment]
The travis failures are only because you're using double quotes instead of single quotes.

(@Hyperkid123 I wonder if that eslint rule is an error intentionally?)

File: scripts/react-gettext-parser-config.js (hunk 2 above)
Comment (mzazrivec) [reply scripts/react-gettext-parser-config.js]: We could, although at the moment we don't use it anywhere in the repo. So I did not add it (yet).

**Comment by Hyperkid123** [comment]
> The travis failures are only because you're using double quotes instead of single quotes.
> 
> (@Hyperkid123 I wonder if that eslint rule is an error intentionally?)

I think it is i did not configure this one. But quotes can be fixed by eslint itself automatically

**Comment by mzazrivec** [comment]
@himdel CI is green now.

**Comment by himdel** [comment]
Aaand merged :)

**Pull request merged**